doc/classes/CollisionShape3D.xml

```diff
@@ -39,6 +39,9 @@
 		<member name="disabled" type="bool" setter="set_disabled" getter="is_disabled" default="false" keywords="enabled">
 			A disabled collision shape has no effect in the world.
 		</member>
+		<member name="physics_material" type="PhysicsMaterial" setter="set_physics_material" getter="get_physics_material">
+			The shape's physics material. Setting a physics material lets the body have different parts with different properties, like the head and handle on a hammer.
```

Comment:
The documentation should explicitly state what happens when both the body and shape have materials. Does the shape's material take priority over the body?

```suggestion
			The shape's physics material. Setting a physics material lets the body have different parts with different properties, such as the head and handle of a hammer.
```